app/views/admin/payout_reports/index.html.slim

```diff
@@ -20,45 +20,32 @@ table.display.table.table-bordered.table-striped
     th ID
     th Type
     th Manual
-    th Expected # of payments
-    th Total # of payments
+    th Expected - actual # of payments
     th # Payments to be paid
+    th # Legit payments missing addresses
     th Amount
-    th Fees
     th Refresh JSON
     th Download
   tbody
     - @payout_reports.each do |report|
       tr.gradeX
         td = report.created_at.strftime("%B %d, %Y, %H:%M")
-        td = link_to report.id, admin_payout_report_path(report.id)
+        td = link_to report.id[0..6], admin_payout_report_path(report.id)
         td = report.final ? "Final" : "Temp"
         td = report.manual ? "Yes" : "No"
-        td = report.expected_num_payments
-        td = report.num_payments
+        td = report.expected_num_payments - report.num_payments
         td = report.num_payments_to_be_paid
+        td = report.potential_payments.to_be_paid.where(address: "").count
```

Comment:
You need to re-write this since `to_be_paid` contains `where.not(address: "")`

Reply:
Thanks, fixed.

```
irb(main):001:0> PotentialPayment.to_be_paid.to_sql
=> "SELECT \"potential_payments\".* FROM \"potential_payments\" WHERE \"potential_payments\".\"uphold_status\" = 'ok' AND \"potential_payments\".\"reauthorization_needed\" = FALSE AND \"potential_payments\".\"uphold_member\" = TRUE AND \"potential_payments\".\"suspended\" = FALSE AND (amount::numeric > 0) AND \"potential_payments\".\"address\" != '' AND \"potential_payments\".\"address\" IS NOT NULL"
irb(main):002:0> PotentialPayment.to_be_paid.unscope(where: :address).to_sql
=> "SELECT \"potential_payments\".* FROM \"potential_payments\" WHERE \"potential_payments\".\"uphold_status\" = 'ok' AND \"potential_payments\".\"reauthorization_needed\" = FALSE AND \"potential_payments\".\"uphold_member\" = TRUE AND \"potential_payments\".\"suspended\" = FALSE AND (amount::numeric > 0)"
```